cmds.go

```diff
@@ -41,6 +50,20 @@ type braviaAPI struct {
 	PSK      string `env:"OFFSCREEN_PSK" help:"Pre-shared key"`
 }
 
+// BeforeResolve runs before environment variable defaults are applied to
+// the kong structs, allowing us to set build-time values for the Bravia
+// host and PSK before looking in the OFFSCREEN_* env vars.
+// BeforeResolve implements the kong.BeforeResolve interface.
+func (b *braviaAPI) BeforeResolve() error { //nolint:unparam // false positive
+	if b.Hostname != "" {
```

Comment:
I'm confused - why would `b.Hostname != ""` be true in BeforeResolve?
(other than manually setting those values, which seems unlikely) 
Could just be my jetlegged brain here...

Reply:
This is due to kong. It sets the default values from the environment during the reset phase, even though the docs says it does it during resolve. I asked Alec on his channel, but have not had a response. I will add a comment to this.

Reply:
Your jetlagged brain is sharper than my dopey one. You are correct - I had the test backwards - it should have been `b.Hostname == ""`. So in honour of your find and standards, I have added tests for this to ensure the build time vars do not override either env or cli specified vars and that they get used when they should. Thank you.